src/java.base/unix/classes/sun/nio/fs/UnixFileSystem.java

```diff
@@ -73,39 +74,17 @@
 
     private final UnixFileSystemProvider provider;
     private final byte[] defaultDirectory;
-    private final boolean needToResolveAgainstDefaultDirectory;
     private final UnixPath rootDirectory;
 
     // package-private
-    UnixFileSystem(UnixFileSystemProvider provider, String dir) {
+    UnixFileSystem(UnixFileSystemProvider provider) {
+        String dir = SystemProps.initProperties().get("user.dir");
```

Comment:
The original code used `StaticProperty.userDir()`.

Reply:
Corrected in 6503dd8d50e6c3302cf9be3e70f43e84f0dcadb4.